app/Base/components/Navbar/index.tsx

```diff
@@ -79,30 +79,29 @@ function Navbar(props: Props) {
 
     const profileUrl = useRouteMatching(routes.myProfile);
 
-    const [logout] = useMutation<LogoutMutation, LogoutMutationVariables>(
+    const [
+        logout,
+        // { loading: logoutLoading },
```

Comment:
why add this just to not use?

Reply:
We will use this later. One of the component cannot be disabled